sdk/python/lib/pulumi/_version.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016-2024, Pulumi Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from semver import VersionInfo
+
+SDK_VERSION = "3.0.0"
```

Comment:
Shouldn't this be "$(VERSION)" to match the sed expression in Makefile?

Reply:
Sed replaces the whole line here, but because I renamed this at some point, there is indeed a mismatch between the name here (`SDK_VERSION`) and the sed expression which expects `VERSION`.

I renamed this to `_VERSION` to indicate that it's private and updated the sed expression https://github.com/pulumi/pulumi/compare/ce1e6a5ecc815144f76df12c5225060401fbf588..886f8ae2c798fdc151b226295937494adafd2ad4

Reply:
```
make build
>>> from pulumi._version import version
>>> print(version)
3.102.1-dev.0
```